### PR TITLE
Add FileHandleTest to the build

### PR DESCRIPTION
**Pull request by majetideepak**
This change includes the FileHandleTest in the build process.
It also improves the FileTest to use TempFilePath and the linkage.

### CODE REVIEW

File: velox/connectors/hive/CMakeLists.txt [hunk 1]
```diff
@@ -16,3 +16,8 @@ add_library(velox_hive_connector OBJECT HiveConnector.cpp FileHandle.cpp)
 
 target_link_libraries(velox_hive_connector velox_connector
                       velox_dwio_dwrf_reader velox_dwio_dwrf_writer file)
+
+add_executable(filehandle_test ./tests/FileHandleTest.cpp)
```
Comment (mbasmanova) [inline velox/connectors/hive/CMakeLists.txt:20]: Let's move this to velox/connectors/hive/tests/CMakeLists.txt and add velox_ prefix to target name or add the file to the existing velox_hive_connector_test

File: velox/connectors/hive/tests/FileHandleTest.cpp [hunk 2]
```diff
@@ -28,14 +27,12 @@ using namespace facebook::velox;
...
 
-  // TODO: use the appropriate test directory.
-  // Use unique name for each process/thread execution to prevent cross
-  // process/thread race condition
   auto pid = getpid();
   auto tid = pthread_self();
-  const std::string filename =
-      "/tmp/test" + std::to_string(pid) + "_" + std::to_string(tid);
-  remove(filename.data());
+  std::stringstream file;
```
Comment (mbasmanova) [inline velox/connectors/hive/tests/FileHandleTest.cpp:32]: perhaps, use velox/exec/tests/TempFilePath.h ?

File: velox/connectors/hive/CMakeLists.txt (hunk 1 above)
Comment (majetideepak) [reply velox/connectors/hive/CMakeLists.txt:20]: Hi Masha, can you point me to the existing velox_hive_connector_test? Thanks.

Comment (majetideepak) [reply velox/connectors/hive/CMakeLists.txt]: Found it. I missed the latest commits.

File: velox/common/file/CMakeLists.txt [hunk 3]
```diff
@@ -15,9 +15,12 @@
 # for generated headers
 include_directories(.)
 add_library(velox_file File.cpp FileSystems.cpp FileSystems.h)
-target_link_libraries(velox_file velox_exception ${GTEST_BOTH_LIBRARIES}
-                      ${GLOG} ${FOLLY_WITH_DEPENDENCIES} ${FMT})
+target_link_libraries(velox_file velox_exception ${FOLLY_WITH_DEPENDENCIES} ${FMT})
```
Comment (majetideepak) [inline velox/common/file/CMakeLists.txt:18]: I found that the FileTest could be improved as well.

**Comment by majetideepak** [comment]
Fixed the format check

**Pull request closed without merging**